Cargo.toml

```diff
@@ -1,10 +1,15 @@
 [workspace]
 default-members = ["crates/fuel-streams"]
 members = [
-    "benchs/*",
-    "crates/*",
+    "benchs/bench-consumers",
+    "benchs/data-parser",
+    "benchs/nats-publisher",
+    "crates/fuel-data-parser",
+    "crates/fuel-streams",
+    "crates/fuel-streams-core",
```

Comment:
Just curious. Why do we need to hardcode these?

Reply:
We don't need I guess; I was testing if the problem with the publish dry run was because of this, and I forgot to rollback to the glob versions